apps/roam/src/components/settings/NodeConfig.tsx

```diff
@@ -327,6 +328,23 @@ const NodeConfig = ({
             </div>
           }
         />
+        <Tab
+          id="node-suggestive-rules"
+          title="Node Suggestive Rules"
```

Comment:
"Node" is redundant here.
Let's just change it to "Suggestive Mode" to be consistent.

We could add direct links to each node's "Rules" in the Global Setting tab, but that can be deffered.
Likely we'll need more info/docs soon regardless.